lib/Service/ExAppOccService.php

```diff
@@ -124,7 +124,7 @@ public function __construct(
 			protected function configure() {
 				$this->setName($this->occCommand->getName());
 				$this->setDescription($this->occCommand->getDescription());
-				$this->setHidden($this->occCommand->getHidden());
+				$this->setHidden(filter_var($this->occCommand->getHidden(), FILTER_VALIDATE_BOOLEAN));
```

Comment:
what is the purpose of it? it is already `int` on input and `int` in the database

Reply:
never mind, this is fine